holoviews/operation/datashader.py

```diff
@@ -81,9 +81,14 @@ class aggregate(ElementOperation):
 
     The bins of the aggregate are defined by the width and height and
     the x_range and y_range. If x_sampling or y_sampling are supplied
-    the operation will ensure that a bin is no smaller than theminimum
+    the operation will ensure that a bin is no smaller than the minimum
     sampling distance by reducing the width and height when the zoomed
     in beyond the minimum sampling distance.
+
+    By default, the PlotSize operator is applied when this operation
```

Comment:
>the PlotSize operator

"the PlotSize stream"

>be set to match the inner dimensions of the plot in which this is used.

We've standardized to the term "linked" would be good if you used that here.